common/sfml_display.h

```diff
@@ -0,0 +1,321 @@
+#pragma once
+
+// BoB robotics includes
+#include "../common/assert.h"
+#include "../common/pose.h"
+#include "../robots/robot.h"
+
+// Third-party includes
+#include "../third_party/units.h"
+
+// SFML
+#include <SFML/Graphics.hpp>
+
+// Standard C includes
+#include <cstring>
+
+// Standard C++ includes
+#include <vector>
+#include <stdexcept>
+
+namespace BoBRobotics {
+using namespace units::literals;
+
+template<typename LengthUnit = units::length::meter_t>
+class SFMLDisplay
```

Comment:
Maybe ``SFMLAgentArena`` or ``SFMLWorld`` would be better names - display suggests to me at least that it's a lower-level thing

Reply:
:+1: